prow/pod-utils/downwardapi/jobspec.go

```diff
@@ -36,7 +36,8 @@ type JobSpec struct {
 	BuildID   string           `json:"buildid,omitempty"`
 	ProwJobID string           `json:"prowjobid,omitempty"`
 
-	Refs kube.Refs `json:"refs,omitempty"`
+	// refs + extra_refs from a prowjob
+	Refs []kube.Refs `json:"refs,omitempty"`
```

Comment:
This is backwards incompatible but can it break things on deployment?

Reply:
any controller depends on the spec env?

Reply:
I think the only issue would be if prow core upgraded before pod-utils did, or vice versa -- these are local to a specific job

Reply:
we always bump prow core together with podutil images, not sure about other users...